Services/Certificate/classes/Placeholder/Values/class.ilObjectCustomUserFieldsPlaceholderValues.php

```diff
@@ -1,35 +1,20 @@
-<?php
+<?php declare(strict_types=1);
 /* Copyright (c) 1998-2019 ILIAS open source, Extended GPL, see docs/LICENSE */
 
 /**
  * @author  Niels Theen <ntheen@databay.de>
  */
 class ilObjectCustomUserFieldsPlaceholderValues implements ilCertificatePlaceholderValues
 {
-    /**
-     * @var array
-     */
-    private $placeholder;
-
-    /**
-     * @var ilCertificateObjectHelper|null
-     */
-    private $objectHelper;
-
-    /**
-     * @var ilCertificateUtilHelper
-     */
-    private $ilUtilHelper;
+    private array $placeholder;
+    private ?ilCertificateObjectHelper $objectHelper;
+    private ?ilCertificateUtilHelper $ilUtilHelper;
```

Comment:
These types are not optional. Only the passed constructor arguments are optional.